oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -78,7 +78,7 @@ int pldm::responder::oem_ibm_platform::Handler::
 
 int pldm::responder::oem_ibm_platform::Handler::
     oemSetStateEffecterStatesHandler(
-        uint16_t entityType, uint16_t entityInstance, uint16_t stateSetId,
+        uint16_t entityType, uint16_t /*entityInstance*/, uint16_t stateSetId,
```

Comment:
why is instance ID commented?

Reply:
so then we can remove that parameter right?

Reply:
yes, lets clean it up if we are not using.